helm/sealed-secrets/templates/service.yaml

```diff
@@ -11,7 +11,8 @@ metadata:
 spec:
   type: {{ .Values.service.type }}
   ports:
-    - port: {{ .Values.service.port }}
+    - name: http
+      port: {{ .Values.service.port }}
```

Comment:
This breaks `kubeseal` until https://github.com/bitnami-labs/sealed-secrets/pull/648 is merged.